pages/blog/graphql-deep-dive-1.mdx

```diff
@@ -0,0 +1,354 @@
+
+While we can have a separate blog on Schema Stitching versus Federation sometime down the line, each have its own set of merits and demerits which you can understand only if you give both a try.
+
+These videos can help get some basics (but a lot has changed since these videos were released especially with GraphQL Tools introducing [**Type Merging**](https://www.graphql-tools.com/docs/stitch-type-merging/)):
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/Vq0ajno-zgw"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+<br />
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/LKQKn1oFXJU"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+**What are some of the GraphQL events/conferences to watch out for?**
+
+Since GraphQL was released, it garnered a huge interest in the community that a lot of conferences, events and meetups are held around the world keeping GraphQL as the main theme. Some of them are:
+
+- [**The GraphQL Summit**](https://summit.graphql.com/)
+- [**GraphQL Conf**](https://graphqlconf.org/)
+- [**Enterprise GraphQL**](https://hasura.io/enterprisegraphql/)
+- [**GraphQL Asia**](https://graphql.asia/)
+- [**GraphQL Galaxy**](https://graphqlgalaxy.com/)
+
+and there are more including meetups like [**these**](https://www.meetup.com/topics/graphql/) and [**these**](https://graphql.org/community/upcoming-events/). You can find most of the previous sessions recorded on Youtube if you search for it.
+
+**How can I contribute to GraphQL and its ecosystem?**
+
+Every bit of help really counts since GraphQL foundation is run by a set of volunteers and its all open source. You can
+
+- Write blogs like this to spread knowledge amongst the community
+- Host meetups, speak in conferences about your experience and evangelize your best way possible.
+- Contribute to the [**GraphQL spec**](https://github.com/graphql/graphql-spec) with your suggestions (Some suggestions may take years to implement even if it is good, so you may need to have a lot of patience for this)
+- Contribute to the ecosystem of tools leveraging GraphQL be it with documentation, tests, features, bug fixes, feedback and what not. It will definitely help.
+- Facing a challenge with GraphQL which has not been solved before? Build your own tooling and contribute it to the community
+- Or if you want to take it to the next level and want to align your entire organization to help the GraphQL foundation, [become its member](https://foundation.graphql.org/join/) and contribute.
```

Comment:
I would also add some points from the bottom of this page: https://the-guild.dev/about-us

Reply:
Sure. Will do.